js/google_ima.js

```diff
@@ -751,6 +760,7 @@ require("../html5-common/js/utils/utils.js");
        */
       var _onInitialPlayRequested = privateMember(function()
       {
+        this.initialPlayRequestTime = new Date().valueOf();
```

Comment:
Although valueOf returns the same thing as getTime, it looks like valueOf is usually reserved for Javascript internals. If there are other changes to be made to this file, can we also change calls to valueOf to getTime?